step.yml

```diff
@@ -23,6 +23,13 @@ inputs:
       description: Your access token for the account that has access to the Bitrise app.
       is_required: true
       is_sensitive: true
+  - verbose_log: "no"
+    opts:
+      title: Enable verbose logging?
+      summary: Enable debug level logging, if set to "yes".
+      description: Enable debug logging, if set to "yes".
+      value_options: ["no", "yes"]
```

Comment:
please use YAML formatted array, and always try to avoid JSON formatted parts in the step.yml. Although you can have pure JSON in your YML, we want to keep our format as YAML standard as possible.

Reply:
It is technically valid yaml: http://yaml-online-parser.appspot.com/